tests/robot-tests/README.md

```diff
@@ -59,137 +58,144 @@ If you intend to run the tests from your local machine, you will also need to cr
 - `.env.preprod`
 - `.env.prod`
  
- You can copy and rename the `.env.example` file in the `robot-tests` directory, replacing the variable values with those for that file's specific environment. The tests rely on these environment variables being set.
-
-
-## How do I run the tests?
+You can copy and rename the `.env.example` file in the `robot-tests` directory, replacing the variable values with those for that file's specific environment. The tests rely on these environment variables being set.
+
+Variables needed to run tests are as follows: 
```

Comment:
I'm going to move this documentation into `.env.example`. As with the `run_test.py` arguments, I think it's better to avoid the duplication so there is less chance of documentation going stale.